services/scabbard/cli/man/scabbard-ns.1.md

```diff
@@ -0,0 +1,43 @@
+% SCABBARD-NS(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**scabbard-ns** — Provides management of Sabre namespaces.
+
+SYNOPSIS
+========
+
+**scabbard ns** \[**FLAGS**\] \[**SUBCOMMAND**\]
+
+DESCRIPTION
+===========
+This command provides management functionality for the Sabre namespaces of a
+scabbard service.
```

Comment:
As in the sabre-ns-create man page, this page could use a brief description of what "a Sabre namespace" is or is used for.

Reply:
Added the same description that I did for the `create` subcommand.